docker.cpp

```diff
@@ -268,7 +284,7 @@ JSON_DOCUMENT Docker::requestAndParse(Method method, const std::string& path, lo
 
         doc.AddMember("success", false, doc.GetAllocator());
         // Forced bool parameter required to prevent ambiguous function call (long appears to cause problems)
-        doc.AddMember("code", (bool)status, doc.GetAllocator());
+        doc.AddMember("code", (int)status, doc.GetAllocator());
```

Comment:
This comment is now at odds with the code.

Reply:
Thanks, will fix